website/content/contributors/getting-started.md

```diff
@@ -109,6 +109,10 @@ It's important to remember that things take some time when it comes to code revi
 
 Many online resources are available to help you learn about Git, pull requests, and issues. Look for online tutorials, documentation, and forums where you can ask questions and get help from other contributors.
 
+#### Understanding ssh keys and Two Factor Authentication
+
+Contributing to open-source projects requires more than just technical know-how. It's also essential to be aware of security measures such as SSH keys and two-factor authentication (2FA). SSH keys provide a secure way to access remote servers and repositories without the need to enter a password every time, ensuring that user authentication is secure. 2FA adds an extra layer of security by requiring a user to provide two forms of identification before accessing an account or system. This might include a password and a one-time code generated by a mobile app, for example. By requiring contributors to use SSH keys and 2FA, open source projects can help prevent unauthorized access to sensitive data, ensuring that contributions are made securely. For new contributors, it's crucial to familiarize themselves with these security measures before diving in. Resources such as the Github documentation on [SSH keys](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) and [2FA](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication) can provide valuable guidance on how to get started.
```

Comment:
A quick copy-edit pass:
```suggestion
Contributing to open-source projects requires more than just technical know-how. Knowing about security measures such as SSH keys and two-factor authentication (2FA) is also essential. SSH keys provide a secure way to access remote servers and repositories without entering a password every time, ensuring that user authentication is secure. 2FA adds an extra layer of security by requiring a user to provide two forms of identification before accessing an account or system. For example, this might include a password and a one-time code generated by a mobile app. By requiring contributors to use SSH keys and 2FA, open source projects can help prevent unauthorized access to sensitive data, ensuring that contributions are made securely. For new contributors, it's crucial to familiarize themselves with these security measures before diving in. Resources such as the GitHub documentation on [SSH keys](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) and [2FA](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication) can provide valuable guidance on how to get started.
```

Reply:
Thank you for the updates